jwst/resample/resample_utils.py

```diff
@@ -303,3 +303,28 @@ def decode_context(context, x, y):
         )
 
     return idx
+
+
+def _xy_range_from_bbox(xmin, xmax, ymin, ymax, shape, bbox=None):
+
+    if (xmin is xmax is ymin is ymax) and xmin in (0, None):
+        xmin = ymin = 0
+        ymax, xmax = np.subtract(shape, 1, dtype=int)
+
+        if bbox:
+            ((x1, x2), (y1, y2)) = bbox
+            if x1 > x2:
+                x1, x2 = x2, x1
+            if y1 > y2:
+                y1, y2 = y2, y1
```

Comment:
Is there any evidence this is possible, i.e. the starting value is larger than the ending value?
astropy.modeling raises a warning if this is the case: https://github.com/astropy/astropy/blob/main/astropy/modeling/bounding_box.py#L84

Reply:
I have no evidence of this. I preserved the existing functionality: https://github.com/spacetelescope/jwst/pull/6041/files#diff-8726cc4bf91b91dd490c35ce61e58c0c840b249b1868fa50cd25a012b13f3d98R383-R386

I do not know why it was implemented this way. Given that `astropy` does not support "inverted" bounding boxes, I agree this should be removed.